repo-info/get_repo_info.py

```diff
@@ -0,0 +1,55 @@
+import argparse
+import datetime
+import pandas as pd
+from tqdm import tqdm
+from github import Github
+
+tqdm.pandas()
+
+parser = argparse.ArgumentParser()
+parser.add_argument('-t', '--github_access_token', help='GitHub access token to overcome API rate limitations')
+parser.add_argument('-f', '--filepath', help='Filepath of .csv file containing repo data')
+parser.add_argument('-c', '--colname', help='Column name in CSV file pertaining to repo URL')
+args = parser.parse_args()
+
+GITHUB_API_RATE_LIMIT = 5000
+FILEPATH, COLNAME, GITHUB_ACCESS_TOKEN = args.filepath, args.colname, args.github_access_token
+REPO_URLS = pd.read_csv(FILEPATH)[COLNAME].unique()
```

Comment:
Can you consider only repos that have at least one detected test. It's not relevant to do this for repos where all tests have some status: `Accepted`, `Opened`, `Deleted`...

Reply:
Can you elaborate on "one detected test"? I am not quite following.

Reply:
One test where the column `Status` is empty string, so the test has been detected but has not been fixed or addressed in any way.

Reply:
Done. Now, script only looks at repositories where all tests in pr-data or py-data have an empty status

Reply:
Is it *all* tests have empty status or *some* (at least one) test has empty status?

Reply:
At least one.